### ISSUE TITLE
Save confirmation box scaling makes exiting impossible

### ISSUE THREAD

**Issue by callybug** [lsp-plugins/lsp-plugins#442]
Hello.

If I wish to overwrite a saved plugin configuration, I get a confirmation window too small to click Yes or No. I then have to kill the process which takes Carla down with it, losing all my session and settings.

![Screenshot_2024-09-20_11-53-06](https://github.com/user-attachments/assets/90829090-8dd0-4e57-84cd-9be2f0d3d2ab)

I thought playing with window/font scaling might make a difference, and it _did_ one time – coincidentally, the first time I took screenshots (below) – but even with 100% scaling for both windows and fonts, I can still get the cropped off confirmation window.

![Screenshot_2024-09-20_11-50-28lsp](https://github.com/user-attachments/assets/0a9de702-1fee-4736-92fd-a6c762102912)
The confirmation window appearing correctly (only once).

If this is entirely not an LSP issue, and it has something to do with Carla or my XFCE settings, I'm open to suggestions (although I don't get this in any other program, Qt or GTK). I'll experiment some more, and avoid overwriting configurations. Thank you for the fantastic suite of plugins.

**Comment by sadko4u** [comment]
I need to test and reproduce the problem. Maybe scaling/font scaling is the reason.

**Comment by sadko4u** [comment]
Tried to quickly reproduce in KDE with the `devel` branch, not possible for me.
Have you tried the `devel` branch? Maybe the problem is already gone.

**Comment by sadko4u** [comment]
Hi!
Just checked XFCE session under recent OpenSUSE Tumbleweed and standalone version of plugins.
Nothing happens as you've mentioned.  Please have a look.

https://www.youtube.com/watch?v=X7zBGV2QWT8

**Comment by sadko4u** [comment]
Hello @callybug. Did something change with latest releases or you're still facing the problem?

**Comment by sadko4u** [comment]
Looks like I met similar problems when designing custom window. Looks like the problem has been fixed in one of current development branches.

**Comment by sadko4u** [comment]
Changes were merged into the `devel` branch.

**Comment by sadko4u** [comment]
Seems to be fixed in 1.2.23.
